syntax = "proto3";

package common;

service CommonService {
  // 基础Demo
  rpc handle (CommonRequest) returns (CommonResponse) {}
}

// 简单请求
message CommonRequest {
  string request_id = 1;
  bytes request = 2;
  uint32 serialize = 3;
}

// 简单响应
message CommonResponse {
  int32 status = 1;
  string response = 2;
}